includes/Admin/Admin_Page.php

```diff
@@ -194,7 +194,8 @@ public function render_page() {
 
 		$selected_plugin_basename = filter_input( INPUT_GET, 'plugin', FILTER_SANITIZE_FULL_SPECIAL_CHARS );
 
-		$categories = Check_Categories::get_categories();
+		$category_labels = Check_Categories::get_category_labels();
+		$categories      = array_keys( $category_labels );
```

Comment:
Not a blocker, but since `$categories` isn't used in the template anymore, we don't need to have it as a variable. You could move the `array_keys( $category_labels )` to the assignment in line 202 instead to avoid the variable.